README.adoc

```diff
@@ -55,7 +55,8 @@ link:https://github.com/keep-network/keep-ecdsa/blob/master/solidity/contracts/a
 === Client Developers
 
 Client developers will be most interested in the link:./main.go[reference Keep
-Go client], as well as the link:docs/rfc/[RFCs] and repo directory structure 👇
+Go client] and link:./CONTRIBUTING.adoc[CONTRIBUTORS file], as well as the
```

Comment:
I mean I'm going to merge this but………………

CONTRIBUTING.adoc
CONTRIBUTORS file

😝😁😆

Reply:
I think it looks *great* <unclear lack of punctuation>